packages/flutter_tools/lib/src/intellij/intellij_validator.dart

```diff
@@ -364,6 +367,7 @@ class IntelliJValidatorOnMac extends IntelliJValidator {
     required UserMessages userMessages,
     required PlistParser plistParser,
     required String? homeDirPath,
+
```

Comment:
what's with this space?

Reply:
Great question 👻 . Fixed, but it already snuck its way into master